test/mix/tasks/thrift.generate_test.exs

```diff
@@ -38,7 +38,7 @@ defmodule Mix.Tasks.Thrift.GenerateTest do
 
   test "specifying an invalid Thrift file" do
     in_fixture fn ->
-      assert_raise Mix.Error, ~r/Error parsing/, fn ->
+      assert_raise Mix.Error, ~r/reserved language keyword/, fn ->
```

Comment:
I think we should still have a test case that covers the "Error parsing" error.

Reply:
That's fair. This particular error is already covered by one of the other test cases, so I've updated the diff with an "Error parsing" case